### DISCUSSION TITLE
How to plot the multiples of a random distribution (MRD).

### DISCUSSION THREAD

**Discussion by 419278981** [mtex-toolbox/mtex#933] [Ask Anything]
Hello! I have got Five-parameter grain boundary distribution, how can I plot the picture like this: 

![5-Figure5-1](https://user-images.githubusercontent.com/32330722/113723888-5f64f200-9724-11eb-8011-d9386affedc8.png)
 
In addition, is it possible to realize the distribution of grain boundaries through a two-dimensional plane cross-section, such as this article "Measuring the Five-Parameter Grain-Boundary Distribution from Observations of Planar Sections"? 
THX for all of you！

**Comment by kilir** [comment]
Hi,
you may have to explain in what form you have your data, otherwise have a look at `Miller/plot`.

With respect to whether it is possible to realize the exercise in the mentioned article; yes, as far as I can see, Mtex has all the basic functions to do such an analysis, however, you probably have to write the script yourself to implement the procedure.

Hope that helps.
Cheers,
Rüdiger

**Reply by 419278981** [reply]
hello，
THX for your reply. 

The data structure includes misorientation, misorientation axis, and crystal plane tilt angle. In fact, I did not find relevant functions in MTEX, and my research on crystallography is not deep enough. so I hope to communicate together

Yours sincerely,
      Yulong Zhu
